allennlp/semparse/domain_languages/domain_language.py

```diff
@@ -243,7 +261,8 @@ def __init__(self,
             if isinstance(getattr(self, name), types.MethodType):
                 function = getattr(self, name)
                 if getattr(function, '_is_predicate', False):
-                    self.add_predicate(name, function)
+                    side_arguments = getattr(function, '_side_arguments', False)
```

Comment:
Did you want the default value to be `None` instead of `False`?

Reply:
Yes I did, thanks, good catch.